src/Sanderling/Sanderling.Exe/App.Motor.cs

```diff
@@ -46,11 +46,6 @@ WindowMotor GetMotor()
 			{
 				var Process = System.Diagnostics.Process.GetProcessById(EveOnlineClientProcessId.Value);
 
-				if (null == Process)
-				{
-					return null;
-				}
-
```

Comment:
GetProcessById never returns nulls. If process is not found the exception will be thrown by it and the code is handling it correctly already there - by suppressing ArgumentException in the following catch block.